doc/ref_impl/cntt-ri2/chapters/chapter01.md

```diff
@@ -29,19 +29,41 @@ In order to promote the quick deployment of the RI and avoid duplicating effort
 <a name="1.1.2"></a>
 ### 1.1.2 Structure of the document
 
-> To be completed
+The document first consolidates the requirements that need to be addressed by this RI in [Chapter 2](./chapter02.md). [Chapter 3](./chapter03.md) then details the requirements that need to be met in order for anyone to be able to install the RI, whether that be into a community lab, their own corporate or personal lab or some other environment such as CNF Testbed or a public cloud provider. An operational runbook is documented in [Chapter 4](./chapter04.md) to aid the deployment of the RI into a chosen environment and includes step-by-step instructions for a selection of installers as well as steps on how to validate the deployment. Finally, [Chapter 5](./chapter05.md) is a placeholder to allow the documentation of any gaps found during the development of this document and the associated artefacts.
 
 <a name="1.1.3"></a>
 ### 1.1.3 Terminology
 
+Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md).
+
 <a name="1.2"></a>
 ## 1.2 Scope
 
+The scope of this document is as follows:
+1. To generate eco-system requirements for the establishment of the RI, including labs, tooling, installers, releases and automation requirements
+1. Provide detailed description file definition for installers, CNF features, and etc..
+1. Provide detailed lab criteria and operations that are generic enough to allow any environment to be used as the "lab".
+1. Provide an operational runbook for the RI, which includes detailed steps for the deployment and configuration of the RI into a chosen lab environment
+1. Provide detailed design for automation of deployment and testing, provide continuous integration and delivery pipeline for the RI
+1. Gap analysis with required actions for existing eco-system within adjacent community projects
+
 <a name="1.3"></a>
 ## 1.3 Reference Implementation Approach
 
-> Describe the approach taken in building the RI (e.g. combined infra/k8s LCM, or sepaarate, etc.)
+The approach taken for the RI has deliberately started by having separate automation stages for the infrastructure on which the RI will be installed, and the RI itself. Meaning, initially, the RI is not looking to have a single installer that can both build out virtual or physical machines **and** build out the Kubernetes and other components.
```

Comment:
The sentence seems to be recursive: it starts by stating that RI has having separate stages (I) for the infrastructure, and then (ii) on which RI will be installed